cmd/installer/root.go

```diff
@@ -1184,6 +1228,51 @@ func (p *PacksInstallationType) downloadPdscFile(pdscTag xml.PdscTag, skipInstal
 	return err
 }
 
+func (p *PacksInstallationType) loadPdscFile(pdscTag xml.PdscTag, timeout int) error {
+	basePdscFile := fmt.Sprintf("%s.%s.pdsc", pdscTag.Vendor, pdscTag.Name)
+	pdscFilePath := filepath.Join(p.LocalDir, basePdscFile)
+
+	pdscURL := pdscTag.URL
+
+	log.Debugf("Loading %s from \"%s\"", basePdscFile, pdscURL)
+
+	pdscFileURL, err := url.Parse(pdscURL)
+	if err != nil {
```

Comment:
We agreed that we should simply ignore an empty url. 
Are we flagging info warning or error if a url is valid but the file constructed pdsc file path does not exist?
I think warning is sufficient.

Reply:
Upps, overseen. I now skip them in the new line 443